java/memory/src/main/java/io/netty/buffer/ArrowBuf.java

```diff
@@ -41,6 +41,17 @@
 
 import io.netty.util.internal.PlatformDependent;
 
+/**
+ * ArrowBuf is the abstraction around raw byte byte arrays that
```

Comment:
We can probably mention that ArrowBuf is a facade over an underlying chunk of memory (UDLE). Essentially ArrowBuf provides APIs to safely interact with underlying memory UDLE and abstracts away the concept of memory address etc.